.travis.yml

```diff
@@ -24,6 +25,10 @@ matrix:
       dist: xenial
       sudo: true
 
+before_install:
+  - docker pull arafato/azurite
```

Comment:
In the future, this may move to an `azure` org on Docker Hub ( https://github.com/Azure/Azurite/issues/94 ), but it hasn't happened yet. Happy with leaving this as is. Just raising awareness.

Reply:
thanks for pointing out